src/posit/connect/content.py

```diff
@@ -189,10 +189,18 @@ def dashboard_url(self) -> str:
     def app_role(self) -> str:
         return self.get("app_role")  # type: ignore
 
+    @property
+    def owner(self) -> dict | None:
+        return self.get("owner")
+
     @property
     def permissions(self) -> Permissions:
         return Permissions(self.config, self.session, content_guid=self.guid)
 
+    @property
+    def tags(self) -> List[dict]:
+        return self.get("tags", [])
```

Comment:
I was looking into fetching tags independently as part of `client.content.get`.

But this is introducing an edge case I hadn't previously considered. We use `@property` to expose dict fields and relationships to other `Resources` (.e.g., content_item.permissions). In this case, `tags` is in conflict because it should be used for both: the dict field and the relationship. 

Let's consider `client.content.get(guid).tags.create('red')`. We could extend the current implementation of `Resources` to allow for this by having `Resources` implement `List[Resource]`. I did a little experimentation and found that it's fairly straightforward. Just requires a bit of modification to how we are using `count`.

Reply:
Eh, scratch that. Not straightforward...